securedrop_client/queue.py

```diff
@@ -136,7 +136,11 @@ def process(self) -> None:
                 session = self.session_maker()
                 job._do_call_api(self.api_client, session)
                 self.pinged.emit()
-            except (RequestTimeoutError, ApiInaccessibleError) as e:
+            except ApiInaccessibleError as e:
+                logger.debug('Job {} raised an exception: {}: {}'.format(self, type(e).__name__, e))
+                self.api_client = None
```

Comment:
flagging: subsequent jobs will still show as pending -

![Screen Shot 2020-02-03 at 11 34 24 AM](https://user-images.githubusercontent.com/7832803/73671595-3f95f200-4679-11ea-9399-a9e288a4da0d.png)

Reply:
I think at this point (since we're quitting the queue threads) we'll want to call `storage.mark_all_pending_drafts_as_failed(self.session)` such that these pending replies display as failed

Reply:
(part of how i'm testing is by logging back in, and the behavior is very odd: some of the bubbles lose their text, unclear if related to this diff though)

Reply:
`storage.mark_all_pending_drafts_as_failed(self.session)` is called in the Controller's logout method, which is called by on_queue_paused, which provides a good separation of responsibilities, but I will try to repro drafts not showing up as failed

i haven't tried repro'ing yet, but it sounds similar to this issue: https://github.com/freedomofpress/securedrop-client/issues/734

Reply:
i was able to repro the pending reply issue so i'll make a fix

Reply:
yeah the bubbles losing text thing I just checked and is happening on master so we can ignore that in this PR